dmipy/distributions/distribute_models.py

```diff
@@ -275,7 +275,8 @@ def set_fixed_parameter(self, parameter_name, value):
 
     def set_tortuous_parameter(self, lambda_perp,
                                lambda_par,
-                               volume_fraction_intra):
+                               volume_fraction_intra,
+                               S0_responses=None):
```

Comment:
I think the S0_responses should not be given as a parameter of the `set_tortuous_parameter` function, but should be given to the `SD1_WatsonDistributedModel` as a second argument, and before a property as `self.S0_responses`, just as in the upper MC-models.

You could, potentially, add a parameter here like `use_S0_debiasing_if_possible` or something to just be able to choose here.

Reply:
For the moment I would opt for `self.S0_tissue_responses` for consistency with other functions.
Do you like `set_tortuous_parameter(..., use_tissue_S0=True)` for triggering it?

Reply:
ah of course keep things consistent. I just meant that I didn't like the multiple places to give `S0_tissue_responses`, which would make it possible to do things differently in different places as the same time.

perfect on the option.

Reply:
See my comment above, we do need to find a way to nicely define which model is associated with what S0 response for intra or extra.

Perhaps we can make the signature even simpler and do `def set_tortuous_parameter(model_intra, model_extra, use_tissue_S0=True)`, and accept the model definitions themselves as input parameters, like


```
stick=C1Stick()
zepp = G2Zeppelin()
watson = SD1WatsonDistributedModel(models = [stick, sepp], S0s=[1, 2])
watson.set_tortuous_parameter(model_intra=stick, model_extra=zepp, use_tissue_S0=True)
```
Internally we can just check for the appropriate model with `isinstance` and get the names for the parameters and associated S0s... what do you think?